quickwit-indexing/src/source/kinesis/api.rs

```diff
@@ -21,13 +21,14 @@
 #![allow(dead_code)]
 
 use rusoto_kinesis::{
-    GetRecordsInput, GetRecordsOutput, GetShardIteratorInput, Kinesis, ListShardsInput, Shard,
+    GetRecordsInput, GetRecordsOutput, GetShardIteratorInput, Kinesis, KinesisClient,
+    ListShardsInput, Shard,
 };
 
 /// Gets records from a Kinesis data stream's shard.
 /// <https://docs.aws.amazon.com/kinesis/latest/APIReference/API_GetRecords.html>
 pub(crate) async fn get_records(
-    kinesis_client: &(dyn Kinesis + Send + Sync),
```

Comment:
I initially wanted to use the `Kinesis` trait to mock the client but since we're testing everything will LocakStack, using the concrete implementation directly everywhere makes everything easier.

Reply:
agreed